packages/firestore/test/unit/specs/limbo_spec.test.ts

```diff
@@ -927,6 +928,48 @@ describeSpec('Limbo Documents:', [], () => {
     }
   );
 
+  // Reproduce the bug in b/238823695
```

Comment:
Please link to the GitHub issue rather than the buganizer ticket. Also, instead of "reproduce" maybe say "avoid regression of" since you've fixed the bug and it's not reproducible anymore!